content/swiftui/concepts/views/terms/toggle/toggle.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: 'Toggle'
+Description: 'Chooses between an "on or off" state based on a Bool type value.'
+Subjects:
+  - 'Software Development'
+  - 'Mobile Development'
+  - 'Computer Science'
```

Comment:
```suggestion
  - 'Mobile Development'
  - 'Computer Science'
```

For now, I'm suggesting we omit `'Software Development'` from here since it isn't on the list in `subjects.md` (this list is curated mostly from the list of available subjects in Codecademy's [course catalog](https://www.codecademy.com/catalog) - towards the bottom left)